File: text-for-java/text4net/text4net-core/TestCore.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using com.gs.text4net.core.parser.csv;
using com.gs.text4net.core.cfg;

namespace com.gs.text4net.core
{
    class TestCore
    {
        public void test()
        {
            CsvParser parser = new CsvConfiguration().configure().
                buildParserFactory().buildParser();

        }
    }
}

File: text-for-java/text4j/text4j-core/src/test/java/com/gs/text4j/core/TestCore.java
package com.gs.text4j.core;

import com.gs.text4j.core.cfg.CsvConfiguration;
import com.gs.text4j.core.parser.csv.CsvParser;

public class TestCore {
	
	public void test() {
		CsvParser parser = new CsvConfiguration().configure()
				.buildParserFactory().buildParser();
		
	}
}
